sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -327,7 +327,7 @@ void gnrc_ipv6_nib_handle_pkt(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
 void gnrc_ipv6_nib_handle_timer_event(void *ctx, uint16_t type)
 {
     DEBUG("nib: Handle timer event (ctx = %p, type = 0x%04x, now = %ums)\n",
-          ctx, type, (unsigned)xtimer_now_usec() / 1000);
+          ctx, type, (unsigned)evtimer_now_msec());
```

Comment:
This might result in different values. Not sure though.

Reply:
It will differ when `xtimer_now_usec()` just has overflown. But I think it is okay ... it's just a debug message.